### PR TITLE
DropDownButton: Popup content is truncated when its width is set by css (T827680)

### PR DESCRIPTION
**Pull request by ksercs**

### CODE REVIEW

File: testing/tests/DevExpress.ui.widgets/dropDownButton.tests.js [hunk 1]
```diff
@@ -13,11 +13,21 @@ const DROP_DOWN_BUTTON_ACTION_CLASS = "dx-dropdownbutton-action";
 const DROP_DOWN_BUTTON_TOGGLE_CLASS = "dx-dropdownbutton-toggle";
 
 QUnit.testStart(() => {
-    const markup = '' +
-        '<div id="container">' +
-            '<div id="dropDownButton"></div>' +
-            '<div id="dropDownButton2"></div>' +
-        '</div>';
+    const markup =
```
Comment (San4es) [inline testing/tests/DevExpress.ui.widgets/dropDownButton.tests.js:16]: Can we add these styles via the `css` method in the test case?

Comment (ksercs) [reply testing/tests/DevExpress.ui.widgets/dropDownButton.tests.js:16]: Yes, thanks

File: testing/tests/DevExpress.ui.widgets/dropDownButton.tests.js [hunk 2]
```diff
@@ -249,6 +249,25 @@ QUnit.module("popup integration", {
 
     });
 
+    QUnit.test("popup width should change if content is truncated", (assert) => {
+        const $dropDownButton = $("#dropDownButton").dxDropDownButton({
+            icon: "square",
+            opened: true,
+            dropDownContentTemplate: function(data, $container) {
+                $("<div>")
+                    .addClass("custom-color-picker")
```
Comment (San4es) [inline testing/tests/DevExpress.ui.widgets/dropDownButton.tests.js:258]:
```suggestion
                    .addClass("custom-color-picker")
                    .css("width:82px; padding:5px;")
```

File: testing/tests/DevExpress.ui.widgets/dropDownButton.tests.js [hunk 3]
```diff
@@ -249,6 +249,25 @@ QUnit.module("popup integration", {
...
+                    .appendTo($container);
+            }
+        });
+
+        const colorPicker = $(".custom-color-picker");
+        colorPicker.css("width:82px; padding:5px;");
```
Comment (San4es) [inline testing/tests/DevExpress.ui.widgets/dropDownButton.tests.js:264]:
```suggestion

```

File: testing/tests/DevExpress.ui.widgets/dropDownButton.tests.js (hunk 2 above)
Comment (ksercs) [reply testing/tests/DevExpress.ui.widgets/dropDownButton.tests.js:258]: ".css" method returns the specified CSS property, so it doesn't work. Css property could not be appended to element by jquery ".appendTo"

**Review by DokaRus — APPROVED** [review]

**Review by alexander-kotov-dx — APPROVED** [review]

**Review by San4es — COMMENTED** [review]
👍

**Pull request merged**